config/etcd-io/org.yaml

```diff
@@ -121,6 +121,15 @@ teams:
     privacy: closed
     repos:
       website: admin
+  maintainers-auger:
+    description: Granted write access to auger
+    members:
+    - jmhbnz
+    - siyuanfoundation
+    - wenjiaswe
+    privacy: closed
+    repos:
+      auger: admin
```

Comment:
Just making sure you meant `admin` here - the use of `admin` vs `maintain` is not consistent across etcd teams (might be worth changing that in the future!)

Reply:
If you did mean admin, I would also recommend reflecting that in the team description by s/write/admin

Reply:
Good catch, I'll confess I had just copied it from the above entry which said `admin`.  I think `maintain` should be fine as that seems to be used elsewhere.  We can probably change website to `maintain` but I would prefer to do that separately.